catalyst_voices/packages/libs/catalyst_cose/lib/src/cose_constants.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:typed_data';
+
+import 'package:cbor/cbor.dart';
+
+/// Holds commonly used tags in COSE.
+final class CoseTags {
```

Comment:
It can be abstract, then constructor is not needed
```dart
abstract final class CoseTags
```

Reply:
If you declare it like you suggested then it's possible to create a subclass defined like this:

```
final class MyClass extends CoseTags {

}
```

My intention was to prevent subclasses from being created, private constructor allows it.